crt_portal/static/sass/_uswds-theme-custom-styles.scss

```diff
@@ -175,6 +177,25 @@ form#report-form {
   width: 320px;
 }
 
+.crt-table {
+  @extend .usa-table;
+  font-family: PublicSans-Regular;
```

Comment:
@LindsayYoung I'm noticing that this `font-family` line and the one below are actually having the opposite of the intended effect; they're making the font Times New Roman-y instead of Public Sans. 

Let's remove these lines and we'll see the intended font. 

Here's more on how fonts are set in USWDS, there's some complexity involved unfortunately: https://designsystem.digital.gov/design-tokens/typesetting/font-family/.

Reply:
Sounds good I will make that fix now

Reply:
OK, I added the bold as its own thing and I removed font family. There was a new merge conflict so I resolved that as well.